assets/parameters.toml

```diff
@@ -471,3 +462,143 @@ d3.bj = {s6=0.3223, a1=0.0, s8=0.0, a2=4.5050, doi="10.1021/acs.jpca.1c01295"}
 
 [parameter.dsdpbep86drpa75]
 d3.bj = {s6=0.3012, a1=0.0, s8=0.0, a2=4.5050, doi="10.1021/acs.jpca.1c01295"}
+
+[parameter.dsdpbep86_km11]
```

Comment:
I would prefer to have a single parametrization per functional rather than many. I don't  want to invent new functional abbreviations here.

Reply:
I would also prefer to have a single parametrization per functional, but the reality is different. For example, MRCC uses these parameters from earlier paper (10.1039/c1cp22592h, see [manual.pdf](https://www.mrcc.hu/MRCC/manual/pdf/manual.pdf), refs 254, 255), some other programs may use later parameters for D3(BJ) corrections. For example, Psi4: https://github.com/psi4/psi4/blob/dfebe5bb4366d3ca9ec2943e6af6ec84136ba749/tests/pytests/data_dft-bench/dftd3-dft-bench-ionization_output.ref#L743-L754.

Reply:
The same situation is for DSD-BLYP. In Psi4 the variant with parameters from 10.1002/jcc.23391 is impemented. Here, in s-dftd3, another set is presented (without any citations, hehe)

Reply:
DSD-BLYP is challenging, the functional was original published as DSD-BLYP with DFT-D1 dispersion ([10.1021/jp1070852](https://doi.org/10.1021/jp1070852)), with GMTKN30 the DSD-BLYP was revised to DFT-D3(0) using the same damping parameters as the D1 version ([10.1021/jp1070852](https://doi.org/10.1021/jp1070852)), but also parameters for DSD-BLYP-D3(BJ) published in the SI of the same publication.